staff/johana-ludwig/blog/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Blog de idiomas y viajes</title>
+</head>
+<body>
+    <header>
+        <h1>Viaja y aprende</h1>
+        <section>
+            <nav>  
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="post.html">Artículo de la semama</a></li>
+                        </ul>
+                    </nav> 
+        </section>
+    </header>
+    <main>
+            <article>
+                <header>
+                    <h2>5 lugares que visitar en Berlín</h2>
+                    <p><small>Autor: <a href="#">J.LO</a></small></p>
+                </header>
+                <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr...<a href="post.html">Ver más</a></p>
+            </article>
+            <br>
```

Comment:
`<br>` tag is not necessary. If you want a space between two `<articles>` you must to use CSS styles

Reply:
ok thank!